docs/infrastructure/index.asciidoc

```diff
@@ -4,83 +4,32 @@
+Alternatively, you can select a single component and view the metrics for that component, or seamlessly switch to view the corresponding logs, application traces or uptime information for that component.
 
-[role="screenshot"]
-image::infrastructure/images/infra-sysmon.jpg[Infrastructure Overview in Kibana]
-
-
-[float]
-== Add data
-
-Kibana provides step-by-step instructions to help you add log data. The
-{infra-guide}[Infrastructure Monitoring Guide] is a good source for more
-detailed information and instructions.
+You can:
 
-[float]
-== Configure data sources 
+* View an inventory of your infrastructure by hosts, Kubernetes pod or Docker containers.
+You can group and filter the data in various ways to help you identify the items that interest you.
 
-The `metricbeat-*` index pattern is used to query the data by default.
-If your metrics are located in a different set of indices, or use a
-different timestamp field, you can adjust the source configuration via the user
-interface or the {kib} configuration file.
+* View current and historic values for metrics such as CPU usage, memory usage, and network traffic for each component.
+The available metrics depend on the kind of component being inspected.
 
-NOTE: Logs and Infrastructure share a common data source definition in
-each space. Changes in one of them can influence the data displayed in the
-other.
+* Use *Metrics Explorer* to group and visualize multiple customizable metrics for one or more components in a graphical format.
+You can optionally save these views and add them to {kibana-ref}/dashboard.html[dashboards].
 
-[float]
-=== Configure source
-
-Configure source can be accessed via the corresponding
-image:logs/images/logs-configure-source-gear-icon.png[Configure source icon]
-button in the toolbar:
+* Seamlessly switch to view the corresponding logs, application traces or uptime information for a component.
 
 [role="screenshot"]
-image::infrastructure/images/infrastructure-configure-source.png[Configure Infrastructure UI source button in Kibana]
-
-This opens the source configuration fly-out dialog, in which the following
-configuration items can be inspected and adjusted:
-
-* *Name*: The name of the source configuration.
-* *Indices*: The patterns of the elasticsearch indices to read metrics and logs
-  from.
-* *Fields*: The names of particular fields in the indices that need to be known
-  to the Infrastructure and Logs UIs in order to query and interpret the data
-  correctly.
-
-[role="screenshot"]
-image::infrastructure/images/infrastructure-configure-source-dialog.png[Configure Infrastructure UI source dialog in Kibana]
-
-TIP: If <<xpack-spaces>> are enabled in your Kibana instance, any configuration
-changes performed via Configure source are specific to that space. You can
-therefore easily make different subsets of the data available by creating
-multiple spaces with different data source configurations.
-
-[float]
-[[infra-read-only-access]]
```

Comment:
To prevent broken links, we typically want to minimize the removal of old IDs. In cases where it's necessary to remove, make sure to add the removed ID to the `redirects.asciidoc` file, along with a link to where the new content lives. Here's an example. The removed ID was named `uptime-security`.

https://github.com/elastic/kibana/blob/06706b574873d5f99aa1485cbbe380709726d86c/docs/redirects.asciidoc#L23-L26

Reply:
OK. This "link" referred to Infrastructure-specific content at such a low level of granularity that I can't imagine anything outside of the Infrastructure content wanting to link specifically to this section. I've checked the rest of the Kibana docs and, as expected, nothing does. I think this link may have been added 

This content is simply a **Tip** now within a section, not even a section, so there isn't a one-to-one redirect to supply. However, for safety, I've added an entry to `redirects.asciidoc`, to redirect to the containing section.

Reply:
Perfect. There probably wasn't anything linking to that section, but still good to add incase some obscure external blog, tutorial, etc., was linking there 🤷‍♂.